build-support/manage_virtualenv.sh

```diff
@@ -26,13 +26,27 @@ function populate() {
     exit_message
 }
 
+# Retrieve a list of all Python dependencies in this project.
+#
+# Previously we used `./pants dependencies --type=3rdparty ::`, but
+# the `--type` option was removed in 2.9.0.dev0. Pants recommends this
+# implementation as an equivalent, and mentions that "in the future
+# there will be a more robust way of querying and filtering
+# dependencies." At that point, this function implementation can
+# likely be simplified.
```

Comment:
is there a link to an issue about this? if not nbd

Reply:
I don't think so... this was basically copied from the deprecation warning I got.